src/main.js

```diff
@@ -146,15 +147,63 @@ Vue.filter('roundThousands', roundThousands)
 
 Vue.component('height-transition', TransitionHeight)
 
+// This is a global mixin used to clean up any
+// references a component may have after it's destroyed.
+// It's a pretty heavy-handed approach to what we're trying
+// to accomplish but Vue doesn't seem to be removing all references
+// itself, which is leading to large memory leaks on data-heavy pages
+// as Apollo polls and when navigating between pages.
+// This could be more elegant and may be fixed by upgrading to Vue 3.
```

Comment:
Note for the reviewer: this will knowingly flood the console with errors when certain components are destroyed. This happens because Vue really doesn't like when you modify some of these reactive props and may even have a `$nextTick` queued after the component is destroyed (or something). The errors are innocuous for now but once this is out and we can test it in lower environments I'll take another pass at cleaning those up and/or narrowing this list.